src/main/java/javaClasses/homework_4/stepan_alvajyan/Calculations.java

```diff
@@ -0,0 +1,24 @@
+package javaClasses.homework_4.stepan_alvajyan;
+
+public class Calculations {
+
+    double getMean(int a, int b) {
```

Comment:
Please change this method to accept varargs like getMean(int ... numbers). In this case you can not use your method for any number of parameters, please change here and in the methods below.